fr-portfolio.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en-gb">
+<head>
+	<link rel="stylesheet" type="text/css" href="normalize.css">
+	<link rel="stylesheet" type="text/css" href="portfolio.css">
+	<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
+	<title>Freddie's Portfolio</title>
+</head>
+<body>	
+ <main>	
+  <nav class="navbar">
+   <a href="#portfolio">Portfolio</a>
+   <a href="#about">About</a>
+   <a href="#contact">Contact</a>
+  </nav>
+	
+  <section class="welcome-banner" >
+   <h1>Freddie's Portfolio</h1>
+  </section>	
+	
+  <article id="about" class="about-alt">
+	<h2> About me </h2>
+		<div class="about-me-container">
+			<figure>
+				<img src="https://via.placeholder.com/350x200">
+			</figure>
+			<div>
+				<p>Hi this is where I should put some information about myself</p>
+			</div>
+		</div>		
+  </article>
+		
+  <section id="portfolio" class="projects">
+   <h2>Portfolio</h2>
+   <div class="portfolio-container"> <!-- Portfolio container left hand start -->
+     <figure>
+		<img src="https://via.placeholder.com/350x200">
+		<figcaption>This is where my project information will go</figcaption>
+	 </figure>
+	 <figure>
+		<img src="https://via.placeholder.com/350x200">
+		<figcaption>This is where my project information will go </figcaption>
+	 </figure>	
+	 </div>  <!-- Portfolio container left hand end -->
```

Comment:
Jist needs to move slightly to the left to be inline with the opening `<div>` tag. The comment on the end should reflect the CSS, i.e `<!-- .class-name -->` or `<!-- #id-name -->`, extra info can be added after but your class names look good so it should be obvious enough if you just state them.

Reply:
I think I have changed this correctly but not sure

Reply:
Yup it's spot on